README

```diff
@@ -3,3 +3,13 @@ This documentation is built with Sphinx (https://www.sphinx-doc.org/en/master/)
 
 See https://obs-ops.lsst.io/project/contributing.html for details on contributing to the documents.
 
+
+pipetask run -j 32 -d "detector IN (0) AND instrument='LATISS' AND \
```

Comment:
I think this was copied here by mistake.

Reply:
Done